components/01-atoms/buttons/twig/button.twig.test.js

```diff
@@ -0,0 +1,61 @@
+import { join } from 'path';
+import { render, Twig } from 'twig-testing-library';
+
+import loadYaml from '../../../../util/loadYaml';
+import setupTwig from '../../../../.storybook/setupTwig';
+
+setupTwig(Twig);
+
+describe('Twig Button', () => {
+  it('can render a button with the standard data', async () => {
+    const { container } = await render(
+      join(__dirname, 'button.twig'),
+      loadYaml(join(__dirname, 'button.yml')),
+    );
+    expect(container).toMatchInlineSnapshot(`
+      <div>
```

Comment:
In Twig, you can remove whitespace in a few ways - [trim](https://twig.symfony.com/doc/2.x/filters/trim.html) being one. Would that help clean these up or does it matter?

Reply:
Whoa, I was wondering if there was a good way to do this!! Trimming whitespace would be a huge upgrade to readability, and would reduce the size of our test/snap files. I'll poke at this!